src/components/Header.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useContext, useState } from 'react';
+import { Todo } from '../types/Todo';
+import { TodosContext } from '../context/TodosContext';
+
+type PropsHeader = {
+};
+
+export const Header: React.FC<PropsHeader> = () => {
+  const { todos, setTodos } = useContext(TodosContext);
+  const [title, setTitle] = useState<string>('');
+
+  const addTodo = (todo: Todo) => {
+    setTodos([todo, ...todos]);
+  };
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    if (title.trim() !== '') {
```

Comment:
```suggestion
    if (!!title.trim()) {
```
pls see[ falsy values](https://developer.mozilla.org/en-US/docs/Glossary/Falsy)